configs/permissions/drift.json

```diff
@@ -1,3 +1,5 @@
 {
     "*": ["*"]
+    "comparisons": ["read"]
```

Comment:
Looks like you're missing a comma here. 🙂 
```suggestion
    "comparisons": ["read"],
```

Reply:
updated here and the line above, thanks